### PR TITLE
#202 Edit mail as new

### PR DESCRIPTION
**Pull request by oubchid**
https://user-images.githubusercontent.com/7097550/108229279-f4456900-713f-11eb-8203-6b533d8f2ff5.mp4

### CODE REVIEW

File: src/esn.inbox.libs/app/services/send-email/email-sending.service.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ angular.module('esn.inbox.libs')
         subjectPrefix: 'Re: ',
         recipients: getReplyAllRecipients,
         referenceIdHeader: INBOX_MESSAGE_HEADERS.REPLY_TO
+      },
+      editAsNew: {
+        subjectPrefix: 'New: ',
```
Comment (fabienmoyon) [inline src/esn.inbox.libs/app/services/send-email/email-sending.service.js:29]: we don't need a prefix for this one ;)

File: src/esn.inbox.libs/app/services/send-email/email-sending.service.js [hunk 2]
```diff
@@ -24,6 +24,11 @@ angular.module('esn.inbox.libs')
...
+        recipients: getReplyRecipients,
```
Comment (fabienmoyon) [inline src/esn.inbox.libs/app/services/send-email/email-sending.service.js:30]: As I know it's more a `getReplyAllRecipients` here

File: src/esn.inbox.libs/app/services/send-email/email-sending.service.js [hunk 3]
```diff
@@ -24,6 +24,11 @@ angular.module('esn.inbox.libs')
...
+        referenceIdHeader: INBOX_MESSAGE_HEADERS.FORWARD
```
Comment (fabienmoyon) [inline src/esn.inbox.libs/app/services/send-email/email-sending.service.js:31]: `FORWARD` no, it's like a new message

File: src/linagora.esn.unifiedinbox/i18n/fr.json [hunk 4]
```diff
@@ -52,6 +52,7 @@
   "Downloading message \"%s\"...": "Téléchargement du message \"{{ messageSubject }}\"...",
   "Draft": "Brouillon",
   "Drafts": "Brouillons",
+  "Edit as New": "Rédaction",
```
Comment (fabienmoyon) [inline src/linagora.esn.unifiedinbox/i18n/fr.json:55]: Editer comme un nouveau mail

**Review by fabienmoyon — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Review by fabienmoyon — CHANGES_REQUESTED** [review]
need changes

**Review by fabienmoyon — APPROVED** [review]

File: src/esn.inbox.libs/app/services/jmap-item/jmap-item-service.js [hunk 5]
```diff
@@ -213,6 +214,12 @@ angular.module('esn.inbox.libs')
       });
     }
 
+    function editAsNew(message) {
+      emailSendingService.editAsNewEmailObject(message.id, session.user).then(function(replyMessage) {
```
Comment (fabienmoyon) [inline src/esn.inbox.libs/app/services/jmap-item/jmap-item-service.js:218]: `replyMessage` to `newMessage`

**Review by fabienmoyon — APPROVED** [review]

**Review by alagane — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Pull request merged**